requirements.txt

```
@@ -26,7 +26,7 @@ opencv-python-headless
 paramiko
 pika>=1.2.0
 Pillow >= 7.2.0
-pydantic==1.*
+pydantic
```

Should we also enforce a minimum requirement here?
```suggestion
pydantic >= 2.0.0
```